rewrite-core/src/main/java/org/openrewrite/config/License.java

```diff
@@ -0,0 +1,31 @@
+package org.openrewrite.config;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public enum License {
+    Apache2("Apache License Version 2.0","https://www.apache.org/licenses/LICENSE-2.0"),
+    MSAL("Moderne Source Available", "https://docs.moderne.io/licensing/moderne-source-available-license"),
+    Proprietary("Moderne Proprietary","https://docs.moderne.io/licensing/overview");
+    private final String fullName;
+    private final String url;
+
+    public static License fromFullName(String name) {
+        for (License license : values()) {
+            if (license.fullName.equalsIgnoreCase(name)) {
+                return license;
+            }
+        }
+        throw new IllegalArgumentException("Invalid license name: " + name);
```

Comment:
Maybe users would like to specify a different license?

Reply:
Yes, that need to be covered. Cheers